mysite/test/factories/user.py

```diff
@@ -0,0 +1,11 @@
+import factory
+from django.contrib.auth.models import User
+
+
+class UserFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = User
+        django_get_or_create = ('username', 'email')
+
+    username = 'john.doe'
```

Comment:
Instead hard-coding the values for the factory, we can make use of `fake-factory` to generate random values each times. Refer the [documentation for `fake-factory`](http://fake-factory.readthedocs.org/en/latest/)
The code might look like,

```python
from faker import Factory as FakerFactory
faker = FakerFactory.create()

class UserFactory(factory.django.DjangoModelFactory):
    class Meta:
        model = User

    username = factory.LazyAttribute(lambda x: faker.user_name())
    email = factory.LazyAttribute(lambda x: faker.email())
```